eZ/Publish/Core/Repository/Strategy/ContentValidator/ContentValidatorStrategy.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Repository\Strategy\ContentValidator;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use eZ\Publish\SPI\Repository\Validator\ContentValidator;
+
+class ContentValidatorStrategy implements ContentValidator
```

Comment:
> Naming is a bit inconsistent with what you've introduced for thumbnails in the past, while the behavior follows similar pattern AFAICS. I mean, for thumbnails you have `ThumbnailChainStrategy`.

Yes, but it isn't the same case. `ThumbnailChainStrategy` has exact meaning: its purpose is to chain various strategies (like `FirstMatchingFieldStrategy`, `StaticStrategy`) and execute them one after another. Here we have one strategy that picks right validator for given object. I don't think Chain or Aggregate is valid here as it would imply this strategy aggregates sub-strategies.